internal/capability/operator_install_test.go

```diff
@@ -0,0 +1,90 @@
+package capability
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/opdev/opcap/internal/operator"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Operator install tests", func() {
+	os.Chdir(os.TempDir())
+
+	csv := operatorv1alpha1.ClusterServiceVersion{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "testcsv",
+			Namespace: "testns",
+		},
+		Status: operatorv1alpha1.ClusterServiceVersionStatus{
+			Phase: operatorv1alpha1.CSVPhaseSucceeded,
+		},
+	}
+
+	fakeClient := operator.NewFakeOpClient(&csv)
+
+	operatorGroupData := operator.OperatorGroupData{
+		Name:             "testog",
+		TargetNamespaces: []string{"default", "testns"},
+	}
+
+	subscription := operator.SubscriptionData{
+		Name:            "testsub",
+		Channel:         "test",
+		InstallModeType: v1alpha1.InstallModeTypeAllNamespaces,
+		Package:         "testpackage",
+		CatalogSource:   "testcatalog",
+	}
+
+	customResources := []map[string]interface{}{
+		{
+			"kind": "testkind",
+			"metadata": map[string]interface{}{
+				"name": "testname",
+			},
+		},
+	}
+
+	auditFn, auditCleanupFn := operatorInstall(context.TODO(),
```

Comment:
This should be either inside the It at line 65, or inside a BeforeEach and remove the one at line 76 if they are not different.

Reply:
They are different, one has a blank namespace given. I was wondering how best to DRY this. Should I just have the initialization happen within each It() and not have initialization inside a BeforeEach?